selfhost/types.jakt

```diff
@@ -1167,6 +1167,8 @@ class CheckedVariable {
     public definition_span: Span
     public type_span: Span?
     public visibility: CheckedVisibility
+    // NOTE: could this be a flag in CheckedVisibility?
```

Comment:
I think it's fine to keep outside, I think visibility should stay semantics-only.